scripts/select_appointments.sql

```diff
@@ -1 +1,20 @@
 -- retrieve appointment information
+SELECT p.given_name, p.surname, a.appt_time
+FROM patients p
+INNER JOIN doctors d
+  ON p.patient_id = d.id
+INNER JOIN appointments a
+  ON d.doctor_id = a.id
+WHERE doctor_id = 6
+;
```

Comment:
Actually, since we don't necessarily know where Dr. Hubbard will be in the database, we should probably look him up by his surname : `WHERE d.surname = 'Hubbard'`